modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -3,10 +3,10 @@
 // * updating/index.adoc
 
 :_content-type: CONCEPT
-[id="Understanding_clusteroperator_conditiontypes_{context}"]
+[id="understanding-clusteroperator-conditiontypes_{context}"]
 = Understanding cluster Operator condition types
 
-The status of cluster Operators includes their condition type, informing you of the current state of your Operator's health. The following definitions cover a list of some common ClusterOperator condition types. Operators that have additional condition types and use Operator-specific language have been omitted. 
+The condition type of cluster Operators informs you of the current state of your Operator's health. The following definitions cover a list of some common cluster Operator condition types. Operators that have additional condition types and use Operator-specific language have been omitted. 
```

Comment:
I realize this is pre-existing language, but:

> Operators that have additional condition types and use Operator-specific language have been omitted.

feels not quite right.  ClusterOperators may define their own condition types, but they also use the standardized types like `Available`, `Progressing`, `Degraded`, and `Upgradeable`.  We are not covering the standardized ClusterOperator condition types and some ClusterVersion-specific types.  We are excluding some operator-specific ClusterOperator condition types.  But the standardized ClusterOperator condition types mean we are still covering all the operators, and we are not omitting operators, just condition types.  Lots of words, but hopefully I'm making sense :).  Also maybe a fine hair to split, so no worries from me if you decide to leave this sentence alone.

Reply:
Example from [this CI run][1]:

```console
$ curl -s https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-aws-ovn-heterogeneous/1627978769123774464/artifacts/ocp-e2e-aws-ovn-heterogeneous/gather-extra/artifacts/clusteroperators.json | jq -c '.items[] | select(.metadata.name == "etcd").status.conditions[]'
{"lastTransitionTime":"2023-02-21T10:54:08Z","message":"NodeControllerDegraded: All master nodes are ready\nEtcdMembersDegraded: No unhealthy members found","reason":"AsExpected","status":"False","type":"Degraded"}
{"lastTransitionTime":"2023-02-21T11:06:27Z","message":"NodeInstallerProgressing: 3 nodes are at revision 8\nEtcdMembersProgressing: No unstarted etcd members found","reason":"AsExpected","status":"False","type":"Progressing"}
{"lastTransitionTime":"2023-02-21T10:47:50Z","message":"StaticPodsAvailable: 3 nodes are active; 3 nodes are at revision 8\nEtcdMembersAvailable: 3 members are available","reason":"AsExpected","status":"True","type":"Available"}
{"lastTransitionTime":"2023-02-21T10:45:52Z","message":"All is well","reason":"AsExpected","status":"True","type":"Upgradeable"}
{"lastTransitionTime":"2023-02-21T10:45:52Z","message":"The etcd backup controller is starting, and will decide if recent backups are available or if a backup is required","reason":"ControllerStarted","status":"Unknown","type":"RecentBackup"}
```

The `etcd` ClusterOperator there is using the usual, standard ClusterOperator condition types, and also adding their own `RecentBackup` operator-specific type.  There's no central standard defining what the `RecentBackup` condition is about, that's just up to the etcd operator.

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-aws-ovn-heterogeneous/1627978769123774464

Reply:
Thank you for the insight! Good call - I've updated to say "Some operator-specific ClusterOperator condition types have been omitted. " :)